src/Microsoft.AspNetCore.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -141,8 +140,8 @@ private void KillProcess(int processId)
             {
                 startInfo = new ProcessStartInfo
                 {
-                    FileName = "pkill",
-                    Arguments = $"-TERM -P {processId}",
+                    FileName = "kill",
+                    Arguments = $"-TERM {processId}",
                 };
```

Comment:
I was looking at this and from the following link it says we need to have a `-` in front of the process id
http://fibrevillage.com/sysadmin/237-ways-to-kill-parent-and-child-processes-in-one-command